utils/cliutils/commandsflags.go

```diff
@@ -1511,6 +1519,9 @@ var commandFlags = map[string][]string{
 	XrCurl: {
 		serverId,
 	},
+	XrAuditNpm: {
+		serverId, path,
```

Comment:
Can we add the regular server flags?
```suggestion
		url, user, password, apikey, accessToken, sshPassPhrase, sshKeyPath, serverId, path,
```